### ISSUE TITLE
coova.org domain in /etc/chilli/functions

### ISSUE THREAD

**Issue by gianluca-saba** [coova/coova-chilli#164]
I was working on Coova installed on a Raspberry Pi. 
Everything works just fine, except that unauthenticated clients were able to access google domains. After some check on the configuration files I've realized that the function writeconfig1 in /etc/functions at the line:
uamallowed	"www.coova.org${HS_UAMSERVER:+,$HS_UAMSERVER}$webadmin$uamallow"
adds automatically the coova.org domain between the allowed hosts. Coova.org seems be hosted by google causing that they share same ip addresses.
in short, to deny access to google to unauthenticated users, I've changed the function line in this way:
uamallowed	"$HS_UAMSERVER$webadmin$uamallow"

Thanks you all,
g

**Comment by sevan** [comment]
@wlanmac as the project is now on github, permitting github.coova.io means permitting access to github by default, can we drop the reference to the coova site as mentioned by @gianluca-saba rather than s/www.coova.org/coova.github.io?

**Comment by wlanmac** [comment]
We should just remove the coova.org walled garden entry...
